Assignments/stu7/data-modeling.sql

```diff
@@ -2,6 +2,52 @@
  * Ex. 11
  */
 
+-- This script was generated by a beta version of the ERD tool in pgAdmin 4.
+-- Please log an issue at https://redmine.postgresql.org/projects/pgadmin4/issues/new if you find any bugs, including reproduction steps.
+BEGIN;
+
+
+CREATE TABLE IF NOT EXISTS public.lemondae_stand
+(
+    lemonade_id serial NOT NULL,
+    size_id integer,
```

Comment:
Looks great, however I think we need to add a day (or date) to indicate when the glass of lemonade has been sold, right?

Reply:
Okay! Fixing now